src/components/forms/convertkit/index.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react'
 
-const FORM_ID = '123456'
+const FORM_ID = process.env.NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM
```

Comment:
i'd consider making this a prop here so it's not so tied to NEXT_PUBLIC and we can pass in this value